en/02_Developer_Guides/03_Forms/Field_types/04_GridField.md

```diff
@@ -365,15 +381,19 @@ class Player extends DataObject
         $fields = parent::getCMSFields();
 
         if($this->ID) {
-            $teamFields = singleton('Team')->getCMSFields();
-            $teamFields->addFieldToTab(
+            $singletonTeam = singleton(Team::class);
+            $teamEditFields = $singletonTeam->getCMSFields();
+            $teamEditFields->addFieldToTab(
                 'Root.Main',
-                // The "ManyMany[<extradata-name>]" convention
+                // The "ManyMany[<extradata-name>]" convention is necessary here, because this will be passed into the GridFieldDetailForm
                 TextField::create('ManyMany[Position]', 'Current Position')
             );
+            // The "ManyMany[<extradata-name>]" convention won't work (and isn't necessary) here, since this isn't passed into the GridFieldDetailForm
```

Comment:
As above, maybe best to simply not mention the simpler syntax?

Reply:
This simpler syntax is the _only_ syntax for summary fields. Hence "won't work"

Reply:
I've made this a little more explicit.